lte/gateway/release/BUILD.bazel

```diff
@@ -369,15 +369,15 @@ pkg_filegroup(
 
 pkg_filegroup(
     name = "feg_python_services",
-    srcs = ["//orc8r/gateway/python:feg_python_services"],
+    srcs = ["//orc8r/gateway/python:magma_python_orc8r_services"],
     prefix = PY_DEST,
     tags = ["no-cache"],
 )
 
 # This target is used to bundle the Python services and scripts,
 # including all runfiles and pip dependencies, used in the FEG
-# integration tests. The tar file can be copied between builder
-# and runtime Docker images.
+# (and CWF) integration tests. The tar file can be copied between
```

Comment:
I put it in brackets because the main purpose of this target is the FEG integ tests. Perhaps a larger refactoring of the Docker setup for FEG and CWF is needed in the future.

Reply:
Makes sense, thanks!